### PR TITLE
gaudi: don't apply patch for 38.2

### PR DESCRIPTION
**Pull request by wdconinc**
The patch for the list include shouldn't apply to 38.2, since it is included in that tag. See https://gitlab.cern.ch/gaudi/Gaudi/-/commit/54b727f08a685606420703098131b387d3026637.

### CODE REVIEW

**Review by alalazo — APPROVED** [review]
Waiting for a maintainer to approve too

**Review by jmcarcell — APPROVED** [review]

**Pull request merged**
